.circleci/config.yml

```diff
@@ -37,18 +37,26 @@ jobs:
       # run tests!
       # - run: npm run unit
 
+    # For local use only
+      - deploy:
+            name: Add AWS credentials
+            command: |
+              #if [ "${LOCALBUILD}" == "true" ];
+                mkdir ~/.aws ; echo $AWS_ACCESS_KEY ; echo $AWS_ACCESS_SECRET ; echo -e "[default]\n$AWS_ACCESS_KEY\n$AWS_ACCESS_SECRET\n" > ~/.aws/credentials
+              #fi
+
     # Deploy DEV 
       - deploy: 
             name: Deploy to DEV S3
             command: |
-              aws s3 sync dist/ s3://flowbase-site-dev/ --delete --region us-east-2 ; echo "${CIRCLE_BRANCH} deployed to DEV."
+              cat ~/.aws/credentials ; aws s3 sync dist/ s3://flowbase-site-dev/ --delete --region us-east-2 ; echo "${CIRCLE_BRANCH} deployed to DEV."
 
     # Deploy QA
       - deploy: 
             name: Deploy to QA S3
             command: |
-              if [ "${CIRCLE_BRANCH}" == "develop" ] || [ "${CIRCLE_BRANCH}" == "master" ];
-                then aws s3 sync dist/ s3://flowbase-site-QA/ --delete --region us-east-2 ; echo "${CIRCLE_BRANCH} deployed to QA."
+              if [ "${CIRCLE_BRANCH}" == "develop" ] || [ "${CIRCLE_BRANCH}" == "master" ] && [ "${LOCALBUILD}" != "true" ];
```

Comment:
if [ "${CIRCLE_BRANCH}" == "develop" ] || [ "${CIRCLE_BRANCH}" == "master" ] && [ "${LOCALBUILD}" != "true" ]

VS 

   if#  **(** [ "${CIRCLE_BRANCH}" == "develop" ] || [ "${CIRCLE_BRANCH}" == "master" ] **# )** && [ "${LOCALBUILD}" != "true" ]

Im not sure in the yml file what the precedence is for && vs || but i think typically && is higher than ||